scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -0,0 +1,471 @@
+<#
+    .SYNOPSIS
+        Powershell utility to streamline running E2E tests for Windows.
+
+    .DESCRIPTION
+        It is used to wrap all related steps to run E2E tests for Windows;
+        it runs clean up, E2E test of given name, and print logs.
+    
+        To get details about parameters, please run "Get-Help .\Run-E2ETest.ps1 -Parameter *"
+        To find out what E2E tests are supported, just run "Get-Help .\Run-E2ETest.ps1 -Parameter TestName"
+        
+        Please ensure that E2E test folder have below folders/files:
```

Comment:
>Please ensure that E2E test folder have below folders/files: [](start = 8, length = 60)

Can you add on the script a step to copy these files? Putting on the description that these are the dependencies won't be very usable. Or add on the script to copy, or add a warning if didn't find.

Reply:
If the script is run by vsts pipeline, it will already copy the artifacts to the appropriate location.  If the script is run manually, we don't know which build of artifacts or even a version edited locally; therefore I didn't make the script to handle this and try to limit the scope of the script.

Error will be thrown if required folder/files are missing when running the script.